app/uploaders/file_uploader.rb

```diff
@@ -18,6 +18,19 @@ class FileUploader < CarrierWave::Uploader::Base
   before :store, :remember_cache_id
   after :store, :delete_tmp_dir
 
+  ##
+  # By default, CarrierWave copies an uploaded file twice, first copying the file into the cache, then copying the file into the store.
+  # For large files, this can be prohibitively time consuming.
```

Comment:
What's the reason for doing this?  Are we seeing this as an issue?   Are these directories on the same file system?

Reply:
There are occasional issues where a very big file fails to upload correctly and/or where the tmp space fills up.  Thought this could potentially help those scenarios by avoiding a seemingly unnecessary big file copy, but I don't have data to back that up.